### PR TITLE
Release 3.1.2

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.1.2 (2019-12-12)
+
+- Fixed the modal component to not place content incorrectly behind other neighboring items. [#73](https://github.com/blackbaud/skyux-modals/pull/73)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: LGTM. Just one suggestions (take it or leave it). Should this be written in the postivive tense? Something like...
`Fixed the modal component to correctly stack children elements inside the modal.`?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Ooh. I like that

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I've changed it to your wording.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.1.2 (2019-12-12)
+
+- Fixed the modal component to correctly stack children elements inside the modal. [#73](https://github.com/blackbaud/skyux-modals/pull/73)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Child instead of children? Only comment I have. Other than that 👍

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I think children is correct here because it is two child elements in relation to each other. Thoughts? @Blackbaud-DeniseCisneros

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md]: I read it as "child" modifying "element", so if it needed to be plural, the modifier would be the same but element would be pluralized.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: OOOOOOHHHHHH. That makes sense. I'll make the change

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Pull request merged**